src/langcheck/eval/ja/reference_free_text_quality.py

```diff
@@ -182,6 +188,100 @@ def _toxicity_local(generated_outputs: List[str]) -> List[float]:
     return toxicity_scores
 
 
+def fluency(generated_outputs: List[str],
+            prompts: Optional[List[str]] = None,
+            model_type: str = 'local',
+            openai_args: Optional[Dict[str, str]] = None) -> EvalValue[float]:
+    '''Calculates the fluency scores of generated outputs. This metric takes on
+    float values between [0, 1], where 0 is low fluency and 1 is high fluency.
+
+    We currently support two model types:
+    1. The 'local' type, where the our fine-tuned model is downloaded from
```

Comment:
I think it's useful to specify "liwii/fluency-score-classification-ja model" or "line-corporation/line-distilbert-base-japanese model" and add the "Ref:" section of the docstring here, since users will usually read the docs for `fluency()` first instead of `_fluency_local()`.